sklearn/utils/tests/test_validation.py

```diff
@@ -853,3 +855,28 @@ def test_check_scalar_invalid(x, target_name, target_type, min_val, max_val,
                      min_val=min_val, max_val=max_val)
     assert str(raised_error.value) == str(err_msg)
     assert type(raised_error.value) == type(err_msg)
+
+
+def test_check_sample_weight():
+    with pytest.raises(ValueError,
```

Comment:
Do you want to group all the exception within the same parametrize test ;)

Reply:
There is only two exceptions here, and the parameters used are not the same, so I'm not convinced that parametrization would improve readability..